certbot/certbot/plugins/dns_test_common.py

```diff
@@ -38,14 +38,12 @@ def assertTrue(self, *unused_args: Any) -> None:
         See
         https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue
         """
-        ...
```

Comment:
Do you have any sense why we had this there in the first place? I can't find evidence of us using it as a sentinel value or anything, but then...why?

Reply:
Based on what I've read, it's an expression that happens to look nice as a placeholder for a function body. 

Style choice by Adrien, who wrote it originally? I don't think there is any side effect to removing it here. Test code anyway.

Reply:
Cool that makes sense. I thought it was usually used in place of `pass` when you need it for some reason, but maybe it's just used even when you don't need to check for it. Seems safe enough to remove.